src/aiortc/contrib/media.py

```diff
@@ -153,10 +155,23 @@ def player_worker(
             if video_first_pts is None:
                 video_first_pts = frame.pts
             frame.pts -= video_first_pts
-
+            print("Video frame %s retrieved: %s" % (str(frame.index), str(frame)))
```

Comment:
please do not add print statements in existing blank lines, rather add it as a new line to preserve readability